components/clientComponents/globals/FormCaptcha/actions.ts

```diff
@@ -61,15 +61,18 @@ export const verifyHCaptchaToken = async (token: string): Promise<boolean> => {
   return true;
 };
 
-// Looks at the success and score to determine a pass or fail. We can tweak the score over time.
-// See https://docs.hcaptcha.com/enterprise#handling-siteverify-responses
+/**
```

Comment:
This whole actions file should probably be removed and added instead to `@libs/validation/hcaptcha`.  This file is not a server action in that it is never called directly from any client or server component and is only called from a Form Filler action.